args.py

```diff
@@ -20,33 +61,37 @@ def parse_args():
                         help='Print all validation predicitons to console')
 
     # main folder for data storage
-    parser.add_argument('--root-dir', type=str, default=None)
+    parser.add_argument('--root-dir', type=str, default='ecg_data')
```

Comment:
I'd rather not attach the agrs file to the ecg dataset

Reply:
Aren't we using it all the time anyway? This file is only for this project, not a generic args parser